.gitignore

```diff
@@ -12,6 +12,7 @@
 data/*.txt
 data/*.ttf
 data/*.sfd
+/docs/
```

Comment:
What is the `docs` directory for?  The generated docs?  I'm not sure we should distribute those in the tarball as opposed to just posting them.